pom.xml

```diff
@@ -84,6 +84,12 @@
             <version>1.9.5</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.skyscreamer</groupId>
+            <artifactId>jsonassert</artifactId>
+            <version>1.5.1</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Test library for asserting JSON string equality ([link](https://github.com/skyscreamer/JSONassert))